docs/ingestion/kinesis-ingestion.md

```diff
@@ -27,19 +27,25 @@ import TabItem from '@theme/TabItem';
   ~ under the License.
   -->
 
-When you enable the Kinesis indexing service, you can configure supervisors on the Overlord to manage the creation and lifetime of Kinesis indexing tasks. Kinesis indexing tasks read events using the Kinesis shard and sequence number mechanism to guarantee exactly-once ingestion. The supervisor oversees the state of the indexing tasks to coordinate handoffs, manage failures, and ensure that scalability and replication requirements are maintained.
+When you enable the Amazon Kinesis indexing service, you can configure supervisors on the Overlord to manage the creation and lifetime of Kinesis indexing tasks.
 
-This topic contains configuration information for the Kinesis indexing service supervisor for Apache Druid.
+The supervisor oversees the state of the indexing tasks to coordinate handoffs, manage failures, and ensure that scalability and replication requirements are maintained.
+
+Kinesis indexing tasks read events using the Kinesis shard and sequence number mechanism to guarantee exactly-once ingestion. 
+
+This topic contains configuration information for the Amazon Kinesis indexing service supervisor for Apache Druid.
 
 ## Setup
 
 To use the Kinesis indexing service, you must first load the `druid-kinesis-indexing-service` core extension on both the Overlord and the MiddleManager. See [Loading extensions](../configuration/extensions.md#loading-extensions) for more information.
 
 Review [Known issues](#known-issues) before deploying the `druid-kinesis-indexing-service` extension to production.
 
-## Supervisor spec configuration
+## `spec`
 
-This section outlines the configuration properties that are specific to the Amazon Kinesis streaming ingestion method. For configuration properties shared across all streaming ingestion methods supported by Druid, see [Supervisor spec](supervisor.md#supervisor-spec).
+This section outlines the configuration properties that are specific to supervisor-led streaming ingestion for Amazon Kinesis.
```

Comment:
I think the original version (without "supervisor-led streaming") was easier to read and comprehend.

Reply:
if you must mention supervisor: "Streaming ingestion supervisors" is easier to read than  "supervisor-led streaming"